src/app/beer_garden/db/mongo/api.py

```diff
@@ -193,6 +201,8 @@ def query_unique(model_class: ModelType, **kwargs) -> Optional[ModelItem]:
         query_set = _model_map[model_class].objects.get(**kwargs)
         return to_brewtils(query_set)
     except DoesNotExist:
+        if raise_missing:
+            raise
```

Comment:
The default in the parameter listing for this is `False`, so for places that are already using this function there's no change.

I added this because in some places I *need* to raise an exception when something isn't found.

Reply:
ok, sounds good to me